### PR TITLE
Add a data_api stack

### PR DESCRIPTION
**Pull request by kenoir**
### What is this PR trying to achieve?

Adds a new terraform stack for the mooted `data.wellcomecollection.org` services.

Part of: 

- https://github.com/wellcometrust/platform/issues/1397

### Who is this change for?

Folk who want to access (and make available) large datasets.

### Have the following been considered/are they needed?

- [ ] Reviewed by @jtweed
- [ ] Deployed new versions
- [ ] Run `terraform apply`.

### CODE REVIEW

File: data_api/main.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module "elasticdump" {
+  source        = "git::https://github.com/wellcometrust/terraform.git//ecs_script_task?ref=v1.0.0"
+  task_name     = "gatling_loris"
```
Comment (alexwlchan) [inline data_api/main.tf]: ???

File: data_api/main.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module "elasticdump" {
+  source        = "git::https://github.com/wellcometrust/terraform.git//ecs_script_task?ref=v1.0.0"
+  task_name     = "elasticdump"
+  app_uri       = "wellcome/elasticdump:latest"
+  task_role_arn = "${module.ecs_elasticdump_iam.task_role_arn}"
+
+  env_vars = [
+    "{\"name\": \"BUCKET\", \"value\": \"${var.elasticdump_config_bucket}\"}",
```
Comment (alexwlchan) [inline data_api/main.tf]: Ick. We should make the module not be this way!

**Comment by alexwlchan** [comment]
We should teach Travis that anything in the data_api folder should be ignored for non-data tasks.

File: data_api/main.tf (hunk 2 above)
Comment (kenoir) [reply data_api/main.tf]: I'm not sure it needs to at a more current version. 

This is really just a placeholder to add _some script task_ in the future.

**Comment by kenoir** [comment]
@jtweed you mentioned you wanted to work on deploying a new service. This might be a good opportunity to do so.

**Pull request merged**